lib/tasks/check_khousheh.rake

```diff
@@ -220,16 +274,17 @@ namespace :check do
                     updated_cluster_attributes[:last_request_date] = (pm.last_seen > cluster.last_request_date.to_i) ? Time.at(pm.last_seen) : cluster.last_request_date
                     updated_cluster_attributes[:fact_checks_count] = cluster.fact_checks_count
                     updated_cluster_attributes[:last_fact_check_date] = cluster.last_fact_check_date
-                    fact_check = pm.claim_description&.fact_check
                     unless pm_fc_mapping[pm.id].blank?
                       updated_cluster_attributes[:fact_checks_count] = cluster.fact_checks_count + 1
                       updated_cluster_attributes[:last_fact_check_date] = pm_fc_mapping[pm.id] if pm_fc_mapping[pm.id].to_i > cluster.last_fact_check_date.to_i
                     end
-                    cpm_items << { project_media_id: pm.id, cluster_id: cluster_id }
+                    cpm_items << { project_media_id: pm.id, cluster_id: cluster.id }
                     # FIXME: Set the center of the cluster properly
                     updated_cluster_attributes[:project_media_id] = cluster.project_media_id || pm.id
                     updated_cluster_attributes[:title] = cluster.title || pm.title
-                    cluster_items << updated_cluster_attributes
+                    # Update cluster
+                    # FIXME: Update clusters in batches
+                    cluster.update_columns(updated_cluster_attributes)
```

Comment:
@caiosba I have a question why do you prefer updating each cluster instead of bulk-update?

Reply:
I don't prefer, but it was not working, so I fixed it the easiest way. I filed a ticket to improve the performance but in a way that it works as expected.